src/java/org/apache/cassandra/repair/AsymmetricRemoteSyncTask.java

```diff
@@ -39,14 +39,14 @@
  */
 public class AsymmetricRemoteSyncTask extends SyncTask implements CompletableRemoteSyncTask
 {
-    public AsymmetricRemoteSyncTask(RepairJobDesc desc, InetAddressAndPort to, InetAddressAndPort from, List<Range<Token>> differences, PreviewKind previewKind)
+    public AsymmetricRemoteSyncTask(SharedContext.RepairContext ctx, RepairJobDesc desc, InetAddressAndPort to, InetAddressAndPort from, List<Range<Token>> differences, PreviewKind previewKind)
     {
-        super(desc, to, from, differences, previewKind);
+        super(ctx, desc, to, from, differences, previewKind);
     }
 
     public void startSync()
     {
-        InetAddressAndPort local = FBUtilities.getBroadcastAddressAndPort();
+        InetAddressAndPort local = ctx.broadcastAddressAndPort();
```

Comment:
Now it seems like the broadcast address is for the "context" - I'd find this clearer if it was `ctx.localBroadcastAddressAndPort`.

Reply:
rather not change as its easier to port from `FBUtilities.getBroadcastAddressAndPort` this way IMO.  Changing the name could be more confusing